static/gsAdmin/components/changePlanAction.tsx

```diff
@@ -269,89 +271,52 @@ class ChangePlanAction extends DeprecatedAsyncComponent<Props, State> {
-        <NavTabs>
-          <li className={activeTier === PlanTier.AM3 ? 'active' : ''}>
-            <a
-              data-test-id="am3-tier"
-              onClick={() =>
-                this.setState({
-                  activeTier: PlanTier.AM3,
-                  billingInterval: MONTHLY,
-                  contractInterval: MONTHLY,
-                  plan: null,
-                })
-              }
-            >
-              AM3
-            </a>
-          </li>
-          <li className={activeTier === PlanTier.AM2 ? 'active' : ''}>
-            <a
-              data-test-id="am2-tier"
-              onClick={() =>
-                this.setState({
-                  activeTier: PlanTier.AM2,
-                  billingInterval: MONTHLY,
-                  contractInterval: MONTHLY,
-                  plan: null,
-                })
-              }
-            >
-              AM2
-            </a>
-          </li>
-          <li className={activeTier === PlanTier.AM1 ? 'active' : ''}>
-            <a
-              data-test-id="am1-tier"
-              role="link"
-              aria-disabled
-              onClick={() =>
-                this.setState({
-                  activeTier: PlanTier.AM1,
-                  billingInterval: MONTHLY,
-                  contractInterval: MONTHLY,
-                  plan: null,
-                })
-              }
-            >
-              AM1
-            </a>
-          </li>
-          <li className={activeTier === PlanTier.MM2 ? 'active' : ''}>
-            <a
-              data-test-id="mm2-tier"
-              onClick={() =>
-                this.setState({
-                  activeTier: PlanTier.MM2,
-                  billingInterval: MONTHLY,
-                  contractInterval: MONTHLY,
-                  plan: null,
-                })
-              }
-            >
-              MM2
-            </a>
-          </li>
-          {this.hasProvisionPermission() && (
-            <li className={activeTier === PlanTier.TEST ? 'active' : ''}>
-              <a
-                data-test-id="test-tier"
-                onClick={() =>
-                  this.setState({
-                    activeTier: PlanTier.TEST,
-                    billingInterval: MONTHLY,
-                    contractInterval: MONTHLY,
-                    plan: null,
-                  })
-                }
-              >
-                TEST
-              </a>
-            </li>
-          )}
-        </NavTabs>
+        <TabsContainer>
+          <Tabs
+            value={activeTier}
+            onChange={tab => {
+              this.setState({
+                activeTier: tab,
+                billingInterval: MONTHLY,
+                contractInterval: MONTHLY,
+                plan: null,
+              });
+            }}
+          >
+            <TabList>
+              {PLAN_TABS.map(tab => (
+                <TabList.Item key={tab.tier} data-test-id={`${tab.tier}-tier`}>
```

Comment:
this would be a good opportunity to remove the `data-test-id` and rely on role-based selectors (`getByRole(“tab”, name)`) instead.